libcxx/test/std/utilities/debugging/breakpoint__dbx.cmd

```diff
@@ -0,0 +1,3 @@
+run
+cont
+quit
```

Comment:
@daltenty Could you please check if the "breakpoint" tests makes sense and if it can be fixed?